<ui version="4.0" >
 <class>XineramaWidget</class>
 <widget class="QWidget" name="XineramaWidget" >
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="ButtonGroup1" >
     <property name="title" >
      <string>Multiple Monitor Support</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="_enableXinerama" >
        <property name="text" >
         <string>Enable multiple monitor virtual desktop support</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_enableResistance" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable multiple monitor window resistance support</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_enablePlacement" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable multiple monitor window placement support</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_enableMaximize" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable multiple monitor window maximize support</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="_enableFullscreen" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Enable multiple monitor window fullscreen support</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3Table" name="headTable" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="vScrollBarMode" >
      <enum>Q3ScrollView::Auto</enum>
     </property>
     <property name="hScrollBarMode" >
      <enum>Q3ScrollView::Auto</enum>
     </property>
     <property name="numRows" >
      <number>2</number>
     </property>
     <property name="numCols" >
      <number>4</number>
     </property>
     <property name="showGrid" >
      <bool>true</bool>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>Q3Table::Multi</enum>
     </property>
     <property name="focusStyle" >
      <enum>Q3Table::SpreadSheet</enum>
     </property>
     <row>
      <property name="text" >
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text" >
       <string>2</string>
      </property>
     </row>
     <column>
      <property name="text" >
       <string>X Coordinate</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Y Coordinate</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Width</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Height</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel2" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Show unmanaged windows on:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="buddy" >
        <cstring>_unmanagedDisplay</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QComboBox" name="_unmanagedDisplay" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_identify" >
       <property name="text" >
        <string>&amp;Identify All Displays</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>431</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="6" />
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kdialog.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
